common/lib/protocol-definitions/src/test/types/validateProtocolDefinitionsPrevious.generated.ts

```diff
@@ -1680,202 +1956,12 @@ use_old_TypeAliasDeclaration_SummaryType(
     get_current_TypeAliasDeclaration_SummaryType());
 
 /*
-* Validate forward compat by using old type in place of current type
-* If breaking change required, add in package.json under typeValidation.broken:
-* "TypeAliasDeclaration_SummaryType.Attachment": {"forwardCompat": false}
-*/
-declare function get_old_TypeAliasDeclaration_SummaryType_Attachment():
-    TypeOnly<old.SummaryType.Attachment>;
-declare function use_current_TypeAliasDeclaration_SummaryType_Attachment(
-    use: TypeOnly<current.SummaryType.Attachment>): void;
-use_current_TypeAliasDeclaration_SummaryType_Attachment(
-    get_old_TypeAliasDeclaration_SummaryType_Attachment());
-
```

Comment:
I think that these types are now not public, and since I am testing the public entrypoint explicitly, they don't show up in the current API, so they're ignored? That doesn't sound quite right though... shouldn't types that are in the previous version but not the current still be tested?

Reply:
Maybe part of the problem is that some of the member types are internal and others arent? From the API report:

```ts
// @public
export namespace SummaryType {
    // @internal (undocumented)
    export type Attachment = 4;
    // @internal (undocumented)
    export type Blob = 2;
    // @internal (undocumented)
    export type Handle = 3;
    // @internal (undocumented)
    export type Tree = 1;
    const Tree: Tree;
    const Blob: Blob;
    const Handle: Handle;
    const Attachment: Attachment;
}
```